EDDiscovery/EDDConfig.cs

```diff
@@ -373,15 +373,15 @@ private void LoadCommanders()
                             EDCommander cmdr = new EDCommander(0,
                                 SQLiteDBClass.GetSettingString("EDCommanderName" + i.ToString(), commanderName, conn),
                                 SQLiteDBClass.GetSettingString("EDCommanderApiKey" + i.ToString(), apikey, conn));
-                            cmdr.NetLogPath = SQLiteDBClass.GetSettingString("EDCommanderNetLogPath" + i.ToString(), null, conn);
+                            cmdr.NetLogDir = SQLiteDBClass.GetSettingString("EDCommanderNetLogDir" + i.ToString(), null, conn);
```

Comment:
Argh, I checked 3 times; I thought I was doing the rename the right-way around to keep the DB setting the same. Or did I rename one item I shouldn't have?